src/components/capture/Create/index.tsx

```diff
@@ -142,67 +143,69 @@ function CaptureCreate() {
+                header: authenticatedRoutes.captures.createNew.title,
                 headerLink:
                     'https://docs.estuary.dev/guides/create-dataflow/#create-a-capture',
             }}
         >
-            <ResourceConfigHydrator>
-                <EntityCreate
-                    title="browserTitle.captureCreate"
-                    entityType={entityType}
-                    draftSpecMetadata={draftSpecsMetadata}
-                    resetState={resetState}
-                    errorSummary={
-                        <ValidationErrorSummary
-                            errorsExist={detailsFormErrorsExist}
-                        />
-                    }
-                    toolbar={
-                        <EntityToolbar
-                            GenerateButton={
-                                <CaptureGenerateButton
-                                    entityType={entityType}
-                                    disabled={!hasConnectors}
-                                    callFailed={helpers.callFailed}
-                                    postGenerateMutate={mutateDraftSpecs}
-                                    createWorkflowMetadata={{
-                                        initiateDiscovery,
-                                        setInitiateDiscovery,
-                                    }}
-                                />
-                            }
-                            TestButton={
-                                <EntityTestButton
-                                    closeLogs={handlers.closeLogs}
-                                    callFailed={helpers.callFailed}
-                                    disabled={!hasConnectors}
-                                    logEvent={CustomEvents.CAPTURE_TEST}
-                                />
-                            }
-                            SaveButton={
-                                <EntitySaveButton
-                                    closeLogs={handlers.closeLogs}
-                                    callFailed={helpers.callFailed}
-                                    disabled={!draftId}
-                                    materialize={
-                                        handlers.materializeCollections
-                                    }
-                                    logEvent={CustomEvents.CAPTURE_CREATE}
-                                />
-                            }
-                        />
-                    }
-                    RediscoverButton={
-                        <RediscoverButton
-                            entityType={entityType}
-                            disabled={!hasConnectors}
-                            callFailed={helpers.callFailed}
-                            postGenerateMutate={mutateDraftSpecs}
-                        />
-                    }
-                />
-            </ResourceConfigHydrator>
+            <EndpointConfigHydrator>
```

Comment:
This compare is rough... I'm assuming it was just the nesting that changed and everything else is kept the same.

Reply:
Yes, that is it. The addition of the endpoint config store hydrator altered the nesting of all components. No changes were made to them.